.circleci/config.yml

```diff
@@ -316,7 +316,7 @@ jobs:
           paths:
             - .ccache/
 
-  linux-parquet-s3-build:
+  linux-parquet-hive-storage-adapters-build:
```

Comment:
@Arty-Maly you have to rename this under `jobs:` on line 21 as well. Looks like CI jobs did not run at all.

Reply:
Is there some file I am missing? I also was confused why there is no circle ci step. I assume it should at least fail?

Reply:
I am surprised as well.
@pedroerp, @mbasmanova can someone with admin access to CircleCI help see why this PR is not invoking CircleCI runs?

Reply:
I don't see any runs here as well https://app.circleci.com/pipelines/github/facebookincubator/velox?filter=all

Reply:
Could that be due to changes to .circleci/config.yml ?

Reply:
The changes to `.circleci/config.yml` look correct and are required for this PR. Is there a way we can get more information?

Reply:
I did not get any runs in a prior revision before touching .circleci/config.yml